src/harness/testcases/grant_testcase.py

```diff
@@ -501,6 +502,79 @@ def test_WINNF_FT_S_GRA_23(self):
       self.assertEqual(resp['response']['responseCode'], 0)
 
   @winnforum_testcase
+  def test_WINNF_FT_S_GRA_25(self):
+    """One request is for a PAL channel and the other for a GAA channel, 
+    no incumbent present in the PAL and GAA frequency ranges used in the two requests.
+
+    The response should be 0 (NO_ERROR)"""
+
+    # Load the Data
+    pal_low_frequency = 3550000000.0
+    pal_high_frequency = 3560000000.0
+    user_id = 'pal_device'
+    device_a = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_c = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_c.json')))
+
+    pal_record = json.load(
+      open(os.path.join('testcases', 'testdata', 'pal_record_0.json')))
+    ppa_record = json.load(
+      open(os.path.join('testcases', 'testdata', 'ppa_record_0.json')))
+    ppa_record, pal_record = makePpaAndPalRecordsConsistent(ppa_record,
+                                                            [pal_record],
+                                                            pal_low_frequency,
+                                                            pal_high_frequency,
+                                                            user_id)
+
+    # Move the Device to a random location in PPA
+    device_a['installationParam']['latitude'], \
+    device_a['installationParam']['longitude'] = getRandomLatLongInPolygon(ppa_record)
+    device_a['userId'] = user_id
+
+    # Register the devices
+    request = {'registrationRequest': [device_a, device_c]}
```

Comment:
Consider using https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/blob/master/src/harness/sas_testcase.py#L57 to save yourself some work.